pyproject.toml

```diff
@@ -1,3 +1,70 @@
+# -------------------------------------------------------------------------------------------------
+# Build system settings
+# -------------------------------------------------------------------------------------------------
+[build-system]
+requires = ["setuptools>=67.7.2", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+include-package-data = true
+platforms = ["MacOS X", "Posix"]
+zip-safe = false
+
+[tool.setuptools.packages.find]
+include = ["argcomplete"]
+namespaces = false # to disable scanning PEP 420 namespaces (true by default)
+
+[tool.setuptools.package-data]
+argcomplete = ["py.typed", "bash_completion.d/python-argcomplete"]
+
+# -------------------------------------------------------------------------------------------------
+# Project Metadata
+# -------------------------------------------------------------------------------------------------
+[project]
+name = "argcomplete"
+version = "3.0.8"
+description = "Bash tab completion for argparse"
+readme = "README.rst"
+requires-python = ">=3.6"
+license = { text = "Apache Software License" }
+authors = [{ name = "Andrey Kislyuk"}, {email = "kislyuk@gmail.com" }]
+maintainers = []
+classifiers = [
+  "Environment :: Console",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: POSIX",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+  "Development Status :: 5 - Production/Stable",
+  "Topic :: Software Development",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Topic :: System :: Shells",
+  "Topic :: Terminals",
+]
+dependencies = ["importlib-metadata >= 0.23, < 7; python_version < '3.8'"]
```

Comment:
Please update this to be conditional on the Python version, as in the original.

Reply:
Did see the `python_version < '3.8'` on the end or do you want it more explicit for the versions?
_`python_version < '3.8'` + the `python_requires=">=3.6"` would give the span all of 3.6 and 3.7_ 

More explicit, could be something like: 
````
dependencies = [
    "importlib-metadata >= 0.23, < 7; python_version == '3.6'",
    "importlib-metadata >= 0.23, < 7; python_version == '3.7'",
] 
````

Reply:
Never mind, I misread the original line, thanks.